### DISCUSSION TITLE
Cannot connect to inspector

### DISCUSSION THREAD

**Discussion by taufiqridha** [isar/isar#707] [Help] [answered]
Hi please somehow i can't connect to Isar inspector,
my pubspec

```
isar: 3.0.0-dev.8 
isar_flutter_libs: 3.0.0-dev.8 
isar_generator: 3.0.0-dev.8
build_runner: any
```
Start with
```
 if (Isar.instanceNames.isEmpty) {
      return await Isar.open(
        [MeModelSchema, PendukungSchema],
        inspector: true,
      );
    }
```
Got this link
<img width="815" alt="Screen Shot 2022-09-22 at 00 30 01" src="https://user-images.githubusercontent.com/1453050/191571932-4496d66f-ed0a-4821-9ca8-6d0bc73eb6bc.png">

but if i opened i didnt see inspector
<img width="1235" alt="Screen Shot 2022-09-22 at 00 30 13" src="https://user-images.githubusercontent.com/1453050/191571957-be87f611-765b-4244-a242-be0e844487dc.png">

I'm using Genymotion on mac. and here's my flutter doctor
<img width="682" alt="Screen Shot 2022-09-22 at 00 31 20" src="https://user-images.githubusercontent.com/1453050/191572074-674a308c-3e73-468c-a7d7-968cfbb26080.png">

**Comment by simc** [comment, marked as answer]
Please use the latest version of Isar. The pre-release Inspector versions are no longer supported

**Reply by taufiqridha** [reply]
Upgraded to
```
isar: 3.0.0-dev.15
isar_flutter_libs: 3.0.0-dev.15
isar_generator: 3.0.0-dev.15
```
and flutter upgrade
it's running now.
Many thanks @leisim for quick response and your passion!

**Reply by simc** [reply]
3.0.0 is the latest version 🙂

**Reply by taufiqridha** [reply]
Whoops yup switch to 3.0.0 and still working, thanks @leisim

**Comment by muddassar-lab** [comment]
Shift to isar 3.0.0 stable version. 3.00-dev.15 had some problems.
